src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -79,23 +79,6 @@ internal static bool GetIsVararg(this Symbol member)
 
 #nullable enable
         internal static bool GetIsNewExtensionMember(this Symbol member)
-        {
-            switch (member.Kind)
-            {
-                case SymbolKind.Method:
-                case SymbolKind.Property:
-                    return member.ContainingSymbol is TypeSymbol { IsExtension: true };
-                default:
-                    throw ExceptionUtilities.UnexpectedValue(member.Kind);
-            }
-        }
-
-        internal static bool GetIsNewExtensionMember(this MethodSymbol member)
```

Comment:
>nternal static bool GetIsNewExtensionMember(this MethodSymbol member) [](http://example.com/codeflow?start=9&length=69)

I do not understand why these overloads a re removed. I intentionally added them to avoid the need to check the symbol kind